src/client/components/Threads.tsx

```diff
@@ -208,3 +231,59 @@ const XIcon = styled(XMarkIcon)({
   width: '16px',
   height: '16px',
 });
+
+const REPLACE = {
+  SendButton: ClackSendButton,
+  Message: Message,
+};
+
+function Threads4({ channel, onOpenThread }: ThreadsProps) {
+  const threadsData = thread.useThreads({
+    filter: { location: { channel: channel.id } },
+    sortDirection: 'descending',
+  });
+
+  const threadsDataReversed = useMemo(() => {
+    return {
+      ...threadsData,
+      threads: [...threadsData.threads].reverse(),
+    };
+  }, [threadsData]);
+
+  const composerOptions: experimental.ThreadsProps['composerOptions'] =
+    useMemo(() => {
+      return {
+        position: 'bottom',
+        groupID: channel.org,
+        location: { channel: channel.id },
+        name: channel.threadName,
+        url: window.location.href,
+        ...(isDirectMessageChannel(channel.id) && {
+          addSubscribers: extractUsersFromDirectMessageChannel(channel.id),
+        }),
+      };
+    }, [channel.id, channel.org, channel.threadName]);
+
+  return (
+    <OpenThreadContext.Provider value={{ onOpenThread }}>
+      <StyledExperimentalThreads
+        threadsData={threadsDataReversed}
+        composerOptions={composerOptions}
+        replace={REPLACE}
+      />
+    </OpenThreadContext.Provider>
+  );
+}
+
+const StyledExperimentalThreads = styled(experimental.Threads)({
+  '&': { overflowY: 'auto' },
+  '& .cord-inline-reply-button': { display: 'none' },
+  '&.cord-threads': { border: 'none', padding: 0 },
```

Comment:
just a q (I never used styled components): why &.cord-threads? Isn't & already cord-threads?

Reply:
The sdk styles override it, but I can just merge both and use && for better specificity